fg-stitch-lib/src/align/aligners/mod.rs

```diff
@@ -626,15 +626,16 @@ impl<'a, F: MatchFunc> SamRecordFormatter<'a, F> {
     pub fn format(
         &self,
         fastq: &FastqOwnedRecord,
-        alignments: &[Alignment],
-        alt_score: Option<i32>,
+        chains: &[Alignment],
+        pre_alignment_score: Option<i32>,
```

Comment:
updated names to be closer than what the represent.  Perhaps we should consider (in another issue), changing the `Alignment` class to be `Chain` or the like?

Reply:
Yes I think `Chain` or `AlignmentChain` would be better.